pkgs/servers/home-assistant/custom-components/omnik_inverter/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, buildHomeAssistantComponent
+, omnikinverter
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "robbinjanssen";
+  domain = "omnik_inverter";
+  version = "2.6.4";
+
+  src = fetchFromGitHub {
+    owner = "robbinjanssen";
+    repo = "home-assistant-omnik-inverter";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-O1NxT7u27xLydPqEqH72laU0tlYVrMPo0TwWIVNJ+0Q=";
+  };
+
+  propagatedBuildInputs = [
+    omnikinverter
+  ];
```

```suggestion
  ];
  
  doCheck = false; # no tests
```

because currently

```
robbinjanssen/omnik_inverter> Running phase: installCheckPhase
robbinjanssen/omnik_inverter> no Makefile or custom installCheckPhase, doing nothing
````

---

makes sense. thx for the input :)